src/AScatterplot.ts

```diff
@@ -790,13 +791,13 @@ abstract class AScatterplot<T> extends EventEmitter {
     this.selectWithTester(tester);
   }
 
-  private showTooltip(canvasPos: [number, number]) {
+  private showTooltip(canvasPos: [number, number], event?: MouseEvent) {
```

Comment:
why optional? I thought it will always be given?

Reply:
right